src/ExpenseTracker.Core/Entities/Transaction.cs

```diff
@@ -6,12 +6,31 @@
 
 namespace ExpenseTracker.Core.Entities
 {
-    [Table("transaction", Schema ="core")]
+    [Table("transaction", Schema = "core")]
     public class Transaction : BaseModel
     {
-        protected Transaction() { }
+        public decimal Amount { get; protected set; }
 
-        public Transaction(Workspace workspace,TransactionCategory transactionCategory,decimal amount,DateTime transactionDate,string type)
+        public string? TransactionProof { get; protected set; }
```

Comment:
@BirajMainali `TransactionProof` does not indicate that it is a reference to a file. A more descriptive name should be used here. Something like `TransactionImage` or `TransactionFile` etc.